community/assessments/projects/kubescape/self-assessment.md

```diff
@@ -0,0 +1,323 @@
+
+##### Synchronizer component
+
+This component is an optional part of the Kubescape Operator. It enables users to replicate the Kubernetes objects in the cluster (somewhat like `rsynch`) to a remote service. It is used for collecting the Kubescape Operator objects by central services monitoring multiple clusters.
+
+#### Detailed overview
+
+For a deeper architectural overview go [here](https://github.com/kubescape/helm-charts/blob/doc/charts/kubescape-operator/README.md#in-cluster-components-overview)
+
+#### Objects
+
+* ApplicationProfile - storing workload level information about container activities (file, process, systemcall, capabilities)
+* ConfigurationScanSummary - summary object containing results of configuration scanning
+* GeneratedNetworkPolicy - object containing proposed network policies for each workload
+* KnownServer - list of IP to host conversion
+* NetworkNeighborhood - storing workload level information about network activities of each container
+* OpenVulnerabilityExchangeContainer - [VEX](https://github.com/openvex) object that was built from the vulnerability scan together with reachability analysis
+* SBOMSyftFiltered - SBOM of running containers filtered by only those that are in use
+* SBOMSyft - SBOM of running containers
+* SeccompProfile - Seccomp profile helper object (converting it to JSONs on each host)
+* VulnerabilityManifest - vulnerability scan result object
+* VulnerabilityManifestSummary - summary object of the vulnerability scans
+* WorkloadConfigurationScan -  workload level configuration scan output
+* WorkloadConfigurationScanSummary - summary of the workload configuration scan
+
+
+### Actions
+
+#### Configuration scans
+
+Configuration scans can be triggered in three ways:
+* Continuous scanning: every new or updated object is scanned, this is achieved by "watching" Kubernetes API objects
+* Scheduled scanning: using pre-configured recurring triggers to scan configurations in the cluster, this is achieved using CronJobs
+* Triggered scanning: Kubescape Operator exposing API to trigger scan of specific objects or the whole cluster
+
+#### Vulnerability scans
+
+Vulnerability scans can be triggered in three ways:
+* Continuous scanning: every new image that are introduced to the cluster is scanned, this is achieved by "watching" Kubernetes API objects
+* Scheduled scanning: using pre-configured recurring triggers to scan for vulnerabilities in the cluster, this is achieved using CronJobs
+* Triggered scanning: Kubescape Operator exposing API to trigger scan of specific image and workloads in the cluster
+
+#### Object storage
+
+In the storage component, full Kubernetes object API is supported with all the CRUD actions.
```

Comment:
mention that list only returns metadata for performance reasons?

Reply:
I don't think it is an important detail in this context